website/plugins/proto-messages/index.js

```diff
@@ -1,7 +1,8 @@
-const visit = require("unist-util-visit");
+// Copyright 2022 Protosaurus Authors
+// Licensed under the Apache License, Version 2.0 (the "License")
```

Comment:
Sorry for this. I'll add `addlicense` checker/writer later.

Reply:
Awesome. Thanks!